scripts/docker/buildpack-deps/Dockerfile.ubuntu.clang.ossfuzz

```diff
@@ -130,11 +143,13 @@ RUN set -ex; \
 # EVMONE
 RUN set -ex; \
 	cd /usr/src; \
-	git clone --branch="v0.11.0" --recurse-submodules https://github.com/ethereum/evmone.git; \
+	git clone --branch="v0.13.0" --recurse-submodules https://github.com/ethereum/evmone.git; \
 	cd evmone; \
 	mkdir build; \
 	cd build; \
-	CXX=clang++ cmake -G Ninja -DBUILD_SHARED_LIBS=OFF -DCMAKE_INSTALL_PREFIX="/usr" ..; \
+	CXX=clang++ cmake .. -G Ninja \
+		-DCMAKE_CXX_STANDARD=20 -DCMAKE_CXX_FLAGS="-stdlib=libc++" \
```

Comment:
```suggestion
		-DCMAKE_CXX_FLAGS="-stdlib=libc++" \
```

Reply:
Note that this is not compiling solidity but evmone, but they indeed already set it in the cmake: https://github.com/ethereum/evmone/blob/dd38ff4fdea410908f397a5c7115c2877e064c05/lib/evmone/CMakeLists.txt#L33